lib/components/viewers/live-stop-times.tsx

```diff
@@ -151,25 +176,55 @@ class LiveStopTimes extends Component<Props, State> {
       return stopTimeComparator(stopTimesA[0], stopTimesB[0])
     }
 
+    const configDepartureWindow = stopViewerConfig?.timeRange / 86400
```

Comment:
Per how the redux state is initialized, `stopViewerConfig.timeRange` should always be defined.